README.md

```diff
@@ -0,0 +1,124 @@
+# rhea-promise
+
+A Promisified layer over rhea AMQP client.
+
+### Examples
+
+#### Sending a message.
+```ts
+import {
+  Connection, Sender, EventContext, Message, ConnectionOptions, Delivery, SenderOptions
+} from "rhea-promise";
+
+import * as dotenv from "dotenv";
```

Comment:
nit: i'd add a comment that this is optional for loading environment configuration from a file